packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.styles.ts

```diff
@@ -0,0 +1,22 @@
+import { IOverflowSetStyles, IOverflowSetStyleProps } from './OverflowSet.types';
+import { IStyleFunction } from '../../Utilities';
+
+export const getStyles: IStyleFunction<IOverflowSetStyleProps, IOverflowSetStyles> = props => {
+  const { className, vertical } = props;
+  return {
+    root: [
+      'ms-OverflowSet',
+      {
+        position: 'relative',
+        display: 'flex',
+        flexWrap: 'nowrap'
+      },
+      vertical && { flexDirection: 'column' },
+      className
+    ],
+    item: {
```

Comment:
move all of the classnames into this file so that users could opt to using the base without any of the styles or classes.